src/general/IWorldEffect.cs

```diff
@@ -36,20 +38,63 @@ public void OnTimePassed(double elapsed, double totalTimePassed)
     {
         var glucose = SimulationParameters.Instance.GetCompound("glucose");
 
+        var totalAmount = 0.0f;
+        var totalChunkAmount = 0.0f;
+
+        var initialTotalDensity = 0.0f;
+        var finalTotalDensity = 0.0f;
+
         foreach (var key in targetWorld.Map.Patches.Keys)
         {
             var patch = targetWorld.Map.Patches[key];
 
+            if (!patch.Biome.Compounds.TryGetValue(glucose, out EnvironmentalCompoundProperties glucoseValue))
+                return;
+
+            totalAmount += glucoseValue.Amount;
+            initialTotalDensity += glucoseValue.Density;
+            totalChunkAmount += patch.GetTotalChunkCompoundAmount(glucose);
+
             // If there are microbes to be eating up the primordial soup, reduce the milk
             if (patch.SpeciesInPatch.Count > 0)
             {
-                if (patch.Biome.Compounds.TryGetValue(glucose, out EnvironmentalCompoundProperties glucoseValue))
-                {
-                    glucoseValue.Density = Math.Max(glucoseValue.Density * Constants.GLUCOSE_REDUCTION_RATE,
-                        Constants.GLUCOSE_MIN);
-                    patch.Biome.Compounds[glucose] = glucoseValue;
-                }
+                var initialDensity = glucoseValue.Density;
+
+                glucoseValue.Density = Math.Max(glucoseValue.Density * Constants.GLUCOSE_REDUCTION_RATE,
+                    Constants.GLUCOSE_MIN);
+                patch.Biome.Compounds[glucose] = glucoseValue;
+
+                var initialGlucose = Math.Round(initialDensity * glucoseValue.Amount +
+                    patch.GetTotalChunkCompoundAmount(glucose), 3);
+                var finalGlucose = Math.Round(glucoseValue.Density * glucoseValue.Amount +
+                    patch.GetTotalChunkCompoundAmount(glucose), 3);
+
+                var localReduction = Math.Round((initialGlucose - finalGlucose) / initialGlucose * 100, 1);
+
+                patch.LogEvent(new LocalizedString("COMPOUND_CONCENTRATIONS_DECREASED",
+                        glucose.Name, string.Format(CultureInfo.CurrentCulture, TranslationServer.Translate(
```

Comment:
Format should not be used here, instead a nested LocalizedString needs to be used, otherwise this substring wouldn't react to translations. I've used this approach before in Thrive, I'm not exactly sure but it might be in the auto-evo results regarding species splitting.